### DISCUSSION TITLE
Captcha Next.js

### DISCUSSION THREAD

**Discussion by yeboJoseph** [solspace/craft-freeform#1696] [Questions & Answers] [answered]
### What happened?

- When fetching formProperties after a submission the captcha object is not included.  I'm not even seeing a g-recaptcha response
- It isn't in here either:  `{site domain}/freeform/form/properties/{formId} ` 
- My graphql query does get captcha properties

### How can we reproduce this?

1. Set up recaptcha V3
2. Create Next form template.
3. Click submit

### Freeform Edition

Pro

### Freeform Version

5.5

### Craft Version

4.13.6

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

4.1.20

**Comment by kjmartens** [comment, marked as answer]
Sorry for the trouble you're experiencing @yeboJoseph,

Some things changed in Freeform 5. This is because captchas are now integrations instead of their own thing.

Please see the documentation here:
https://docs.solspace.com/craft/freeform/v5/headless/graphql/#form-captcha-interface
https://docs.solspace.com/craft/freeform/v5/headless/graphql/#how-to-render-a-form

If you still require assistance, please let me know. 🙂

**Reply by yeboJoseph** [reply]
Instead of using the response I got the captcha info with graphql, but it looks like it was [resolved in this ticket](https://github.com/solspace/craft-freeform/issues/1774).
